channels/timed_set.go

```diff
@@ -390,3 +390,31 @@ func TimedSetFromString(encoded string) TimedSet {
 	}
 	return set
 }
+
+type CollectionByTimedSet map[uint32]TimedSet
+
+// AtSequenceByCollection creates a map of collectionID to a map of channel IDs to sequences.
```

Comment:
`AtSequenceByCollection` is the constructor creating sequence IDs for each channel.

Reply:
I only see test usages today, so I don't think it's a generic constructor?  I'd like to avoid accidentally using this in production (in general I think it would make more sense to build the TimedSetByCollectionID in the first place, instead of building a channels.Set then converting to TimedSetByCollectionID), unless you think there's an actual use case I'm overlooking.

Reply:
We use this in `db/changes.go`.